README.md

```diff
@@ -79,70 +94,69 @@ and build the project:
 $ git clone git@github.com:simplex-chat/simplex-chat.git
 $ cd simplex-chat
 $ stack install
-$ dog-food
 ```
 
-## Running the chat client
+## Usage
+
+### Running the chat client
+
+To start the chat client, run `dog-food` from the terminal (as in [eating your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food)).
```

Comment:
I'd move explanation back closer to `dog-food`, right now it hangs there after `terminal` implying it elaborates on the whole sentence..